TAMULib/vufind
# Remove Underlining from Links

# Description

Issue: Dotted lines appear beneath links (both images and text). This is caused by the CSS for the `a` and `a:hover` elements including styling for the `border-bottom` property.

This PR includes an update to compiled.css which comments out the bottom-border styling so that links have a cleaner appearance.

## Type of change

Please delete options that are not relevant.


# Testing Procedures

This change is live on [catalog-dev](https://catalog-dev.library.tamu.edu). No underlining should be visible on any links on the home page, search results, or record pages.

Example record page: https://catalog-dev.library.tamu.edu/Record/in00000000002
